xml/System.Diagnostics/Process.xml

```diff
@@ -703,23 +703,23 @@ process.BeginOutputReadLine();
+The <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> property suggests whether the component should be notified when the operating system has shut down a process. The <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> property is used in asynchronous processing to notify your application that a process has exited. To force your application to synchronously wait for an exit event (which interrupts processing of the application until the exit event has occurred), use the <xref:System.Diagnostics.Process.WaitForExit%2A> method.  
   
 > [!NOTE]
->  If you are using Visual Studio and double-click a <xref:System.Diagnostics.Process> component in your project, an <xref:System.Diagnostics.Process.Exited> event delegate and event handler are automatically generated. Additional code sets the <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> property to `false`. You must change this property to `true` for your event handler to be executed when the associated process exits.  
+> If you are using Visual Studio and double-click a <xref:System.Diagnostics.Process> component in your project, an <xref:System.Diagnostics.Process.Exited> event delegate and event handler are automatically generated. Additional code sets the <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> property to `false`. You must change this property to `true` for your event handler to be executed when the associated process exits.  
   
- When an associated process exits after being shut down by the operating system either through a normal or abnormal termination, the operating system notifies each process component to which the process was associated, as long as the component's <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> value is `true`. If a component started the process, the component can then access the administrative information for the associated process, which is still being stored by the operating system. Such information includes the <xref:System.Diagnostics.Process.ExitTime%2A>, and the <xref:System.Diagnostics.Process.ExitCode%2A>.  
+When an associated process exits after being shut down by the operating system either through a normal or abnormal termination, the operating system notifies each process component to which the process was associated if the component's <xref:System.Diagnostics.Process.EnableRaisingEvents%2A> value is `true` (and possibly also when its value is `false`). If a component started the process, the component can then access the administrative information for the associated process, which is still being stored by the operating system. Such information includes the <xref:System.Diagnostics.Process.ExitTime%2A>, and the <xref:System.Diagnostics.Process.ExitCode%2A>.  
```

Comment:
This statement is very confusing, can it be made as, EnableRaisingEvents value is true, or when EnableRaisingEvents is false, and HasExited check was invoked by the component?